e2e/README.md

```diff
@@ -0,0 +1,12 @@
+# End2End Testing
+All e2e tests use [k3s](https://github.com/rancher/k3s) and the build container used in the make process using dapper will install k3s and run all tests. Use `make` to build/run evertything and `make e2e` to target only the e2e tests. The e2e tests require you make ahve run `build-controller` to have the `./bin/terraform-controller` binary available to run tests against.
```

Comment:
ahve > have

Reply:
goodness, this whole thing is bad. i shoulda proofed it better. making more changes than just this.